posts/tests.py

```diff
@@ -1,3 +1,88 @@
-# from django.test import TestCase
+import pytest
+from .models import Poll, Post
+from django.contrib.auth.models import User
 
-# Create your tests here.
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+DESCRIPTION = "this is a test poll"
+FILE1 = "Alon_Shakaroffs_resume.pdf"
+FILE2 = "Olive.png"
+CHOICETEXT1 = "First option"
+CHOICETEXT2 = "Second option"
+
+
+@pytest.fixture
+def new_user():
+    return User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, password=PASSWORD, email=EMAIL)
+
+
+@pytest.fixture
+def new_poll(new_user):
+    return Poll(author=new_user, description=DESCRIPTION)
+
+
+# -------------------------------------------- Poll tests --------------------------------------------
+
+
+# Check that the Poll values are the same as the Poll inputs.
+def test_new_poll(new_poll):
+    assert new_poll.author.username == USERNAME
+    assert new_poll.description == DESCRIPTION
+
+
+# Check if the Poll can be referred as post.
+@pytest.mark.django_db()
+def test_poll_is_post(new_poll):
+    assert len(Post.objects.all()) == 0
+    new_poll.author.save()
+    new_poll.save()
+    assert len(Post.objects.all()) == 1
+    post = Post.objects.all().first()
+    assert hasattr(post, 'poll') is True
+    assert hasattr(post, 'resume') is False
```

Comment:
```suggestion
    assert hasattr(post, 'poll')
    assert not hasattr(post, 'resume')
```
It is redundant to check if True or False